src/api/publicationApi.ts

```diff
@@ -99,16 +96,16 @@ export const getPublication = async (id: string, dispatch: Dispatch) => {
   }
 };
 
-export const lookupDoiTitle = async (url: string) => {
+export const getPublicationByDoi = async (doiUrl: string) => {
   try {
     const idToken = await getIdToken();
-    const response = await Axios.get(`${PublicationsApiPaths.DOI_LOOKUP}${url}`, {
+    const response = await Axios.get(`${PublicationsApiPaths.DOI_LOOKUP}/?url=${doiUrl}`, {
```

Comment:
skal `/` være med i requesten når det er en query?

Reply:
den er optional
https://stackoverflow.com/a/42193734

Reply:
stemmer for å droppe den